components/StyledButton.js

```diff
@@ -15,4 +12,14 @@ export const StyledButton = styled.button`
   &:active {
     box-shadow: 2px 2px 5px indigo;
   }
+  &.generate {
+    position: absolute;
+    bottom: 0.5rem;
+    right: 0.5rem;
+  }
```

Comment:
you can work with props here for consistency (don't forget to import { css })

```suggestion
  &.generate {
    position: absolute;
    bottom: 0.5rem;
    right: 0.5rem;
  }
```

```suggestion
    ${(props) =>
    props.generate &&
    css`
      position: absolute;
      bottom: 0.5rem;
      right: 0.5rem;
    `}
  }
```